kinto/core/resource/schema.py

```diff
@@ -305,12 +372,38 @@ class JsonPatchBodySchema(colander.SequenceSchema):
     operations = JsonPatchOperationSchema(missing=colander.drop)
 
 
+# Request schemas
+
+
 class RequestSchema(colander.MappingSchema):
-    """Baseline schema for kinto requests."""
+    """Base schema for kinto requests."""
+
+    header = HeaderSchema()
+    querystring = QuerySchema()
+
+    def bind(self, header=None, querystring=None, **kwargs):
+        binded = super(RequestSchema, self).bind(**kwargs)
+        if header:
+            binded['header'] = header.bind(**kwargs)
```

Comment:
Do we really want to support that `.bind(**kwargs)`. I find it cool and flexible, but it looks like we don't use it and it does not ease the understanding of the code.